MARDEK Engine/Assets/Scripts/MARDEK/Inventory/Item/JSONReader2.cs

```diff
@@ -0,0 +1,55 @@
+using UnityEngine;
+using UnityEditor;
+using FullSerializer;
+using MARDEK.Core;
+using MARDEK.Stats;
+using MARDEK.Skill;
+using System.Collections.Generic;
+
+namespace MARDEK.Inventory
+{
+    public class JSONReader2 : MonoBehaviour
```

Comment:
Similar to the other importer, this could be named ElementImporter

Reply:
That's weird, I deleted that one a long time ago, how did it make it into the pull request?